index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags always come first -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Buy a Car!</title>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.5/css/bootstrap.min.css" integrity="sha384-AysaV+vQoT3kOAXZkl02PThvDr8HYKPZhNT5h/CXfBThSRXQ6jW5DO2ekP5ViFdi" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header class="page-header container-fluid">
+<!-- 4. Put a standard Bootstrap navbar element at the top of your page. -->
+      <nav class="navbar navbar-light bg-faded row">
+        <a class="navbar-brand col-md-3" href="#">
+          <h1>Otto's Autos</h1><small>and Vinnies Vans</small>
+        </a>
+        <ul class="nav navbar-nav col-md-3">
+          <li class="nav-item">
+            <a class="nav-link" href="#">Some Cars!<span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Some More Cars!</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Email Us!</a>
+          </li>
+        </ul>
+<!-- 5. Put a text input field in the navigation bar. -->
+        <input class="col-md-5" type="text" placeholder="Please update the description..." id="descText">
+      </nav>
+    </header>
+
+    <div class="container" id="insert">
```

Comment:
The layout is a bit claustrophobic with the cards never quite fitting in here all at once. Try giving them a little breathing room at the top and bottom